tests/components/reolink/test_config_flow.py

```diff
@@ -313,29 +314,66 @@ async def test_reauth(hass):
         data=config_entry.data,
     )
 
-    assert result["type"] == "form"
+    assert result["type"] == data_entry_flow.FlowResultType.FORM
     assert result["step_id"] == "reauth_confirm"
 
     result = await hass.config_entries.flow.async_configure(
         result["flow_id"],
         {},
     )
 
-    assert result["type"] == "form"
+    assert result["type"] == data_entry_flow.FlowResultType.FORM
     assert result["step_id"] == "user"
     assert result["errors"] == {}
 
     result = await hass.config_entries.flow.async_configure(
         result["flow_id"],
         {
-            CONF_HOST: TEST_HOST2,
             CONF_USERNAME: TEST_USERNAME2,
             CONF_PASSWORD: TEST_PASSWORD2,
         },
     )
 
-    assert result["type"] == "abort"
+    assert result["type"] == data_entry_flow.FlowResultType.ABORT
     assert result["reason"] == "reauth_successful"
-    assert config_entry.data[CONF_HOST] == TEST_HOST2
+    assert config_entry.data[CONF_HOST] == TEST_HOST
     assert config_entry.data[CONF_USERNAME] == TEST_USERNAME2
     assert config_entry.data[CONF_PASSWORD] == TEST_PASSWORD2
+
+
+async def test_dhcp_flow(hass):
```

Comment:
We should add a test for avoiding duplicate entries.

E.g., test an existing config entry (e.g., user originated), some device comes in from DHCP and is thus ignored.

Reply:
@frenck I just added a extra test to check dhcp aborts if the entry already exists.
Thanks for suggesting this!
Could you approve now?

Reply:
> Could you approve now?

Lol, you keep asking that, no need to ask that. 
Sometimes stuff will get multiple reviews.